README.md

```diff
@@ -156,6 +156,33 @@ export const UserEchoWithIntervalComponent = () => {
 };
 ```
 
+Besides that, you can also perform calls manually, and check the current amount of times your request was performed.
+`load` function is a promise loader function from the `usePromise` hook.
+
+```tsx
+export const IntervalAndManualCheckComponent = () => {
+    const echoWithArguments = (param: string) => echo(param);
+    const [result, start, stop, load,, retries] = usePromiseWithInterval<string, [string]>(echoWithArguments, 2000);
```

Comment:
needless double comma: `load,,`

Reply:
nope, there's a `reset` method there, not used. That's how you omit it

Reply:
resolved, I described all new (and some old) values that are being returned.